cymetric/root_metrics.py

```diff
@@ -56,6 +56,7 @@ def root_metric(obj=None, name=None, schema=None, *args, **kwargs):
 agent_exit = root_metric(name='AgentExit')
 transactions = root_metric(name='Transactions')
 info = root_metric(name='Info')
+time_step_dur = root_metric(name='TimeStepDur')
```

Comment:
Maybe I'm missing something, but I don't see how this name matches a dependency of the TimeList above?